kubernetes.yml

```diff
@@ -0,0 +1,133 @@
+        echo "export FLANNEL_NET=192.168.0.0/16" >>/root/kubernetes/cluster/ubuntu/config-default.sh
+        sed -i "s/,DefaultStorageClass//" /root/kubernetes/cluster/ubuntu/config-default.sh
+
+
+        #Set Private Key
+        mkdir /root/.ssh
+        echo "-----BEGIN RSA PRIVATE KEY-----" >>/root/.ssh/id_rsa
+        echo "MIIEpAIBAAKCAQEAynFinbJGW2Sg+BLE0OpsNFC0S6CLfoy5P8auJzFHxSdC+uU7" >>/root/.ssh/id_rsa
+        echo "BK1GGcpZcB57Ed1iOLLLoR/1GseyNTMKefGD6Uee0lqc9yrAyUicIab/W7rPG+1g" >>/root/.ssh/id_rsa
+        echo "rwR6a+bT8rqv8C/aRWl5NfohibJIgaepMCJI51kxdW9jzHmwNB18xaekPd/6yIhY" >>/root/.ssh/id_rsa
+        echo "28N+fg+EDMStDOHcqZWpW91en64MYN7AjKccrgPp+OES40XEXartD9camhhVnhfV" >>/root/.ssh/id_rsa
+        echo "C4BkgmlKs6zQII8A4k6LhAjGFimqMq9mYc8VL1b4M+1TE9eX21FllNhSF7y8JrpV" >>/root/.ssh/id_rsa
+        echo "6r3wOhq3msDGtHq3m9c25CJ/OfiiCbAUE8g8XwIDAQABAoIBAQC7fTdwBWPT0Oaj" >>/root/.ssh/id_rsa
+        echo "2f4d3n5IFVZllFKc6YfH4SH5elKV7T7DTvnR4whBan5asGAas4X8LNsaZpaCLuCF" >>/root/.ssh/id_rsa
+        echo "FLhnJ56btpEJEJK+9wGFaZHwONHXCrQ1kSsx7FvVAvrYmr3HHQpxrrVroDTfahZs" >>/root/.ssh/id_rsa
+        echo "vfKbeaU7XgfIU1mCTlE6cvTTuecKxXTXENDFyxNb1bPU5UKMiqY61N0a6xpqTP1Z" >>/root/.ssh/id_rsa
+        echo "3lVfuuGm+BPEWQ2ssj21WOIJgh0gWQunrOT9CMBjW1VKlrLvwa3IQ6DrUIa7zuAg" >>/root/.ssh/id_rsa
+        echo "QcaKXZONXr02LOpQ1rOvNAB2X8WqQ3pf9kfslaEbY9zh0tr1vLScvXQplRnlYhYI" >>/root/.ssh/id_rsa
+        echo "caC16XqBAoGBAP/aL5R9xTJUx6G9uMQJ4g0AfvlkA1Oi+67Xv7S5U85CV38K01ax" >>/root/.ssh/id_rsa
+        echo "so1DuB4DaoZVdMILBrrJjn9GXRHKELzKdUoz1b5qbuNGRLmkY1giJUOVvKovMGXl" >>/root/.ssh/id_rsa
+        echo "6+WysTyX2Sa1bXW4Mo2nwwHQo48BLAmrShmbZ86rk0uBU9jPyGujUrCHAoGBAMqP" >>/root/.ssh/id_rsa
+        echo "Tj2BrRNpdBZckor3IkiizLEMgDuwCZ+bbXczUfNpK1fhezgafvW+t1usv45rnl0C" >>/root/.ssh/id_rsa
+        echo "wBGLRpCXrNTGd62trM4XGYJMcbDSb12/lgXln14WxpAzDoMr9ZjTzLwPLsMliR96" >>/root/.ssh/id_rsa
+        echo "pBecTi6voLGaFpdGs+5NJ9Ywn+xhXsp2xtsfIcNpAoGAK9ZZgqrOB+Ko1dio+MXs" >>/root/.ssh/id_rsa
+        echo "umeUWey9+S2azAlhani6mEObtEPJlDgcLMImoNXbAatQtxKmk4iHZt1KgwrJNV/H" >>/root/.ssh/id_rsa
+        echo "GvCotdiMqVpX2isFbotjMSJN0icFtt/fPFsSrRwjc9NwLtRhrQ3KhtwVuiHsRrfX" >>/root/.ssh/id_rsa
+        echo "mW2cKrgRI8XD9QkdAPxZ5a8CgYEAtW/9xiw0qpGn0Svke6N4SxXAQcqPxmcdh6DC" >>/root/.ssh/id_rsa
+        echo "QaZvfpeyokovhAZ/uIG3Xd9zyg9tz6RaBJz7fSLPRdvscV6Czit0xvjQUXF9Hb0x" >>/root/.ssh/id_rsa
+        echo "lWCkCrRd8eY51pb2q4jz93OmEFi75b2F5Dw276yTROJf3+sN2qOzzjDJ3wMs1C/m" >>/root/.ssh/id_rsa
+        echo "WXoeVRkCgYBAFdc+bDrA6Bn6pjJgpayL+22dXuGrkJgXiwb0hO+r9/LKuckJ3la0" >>/root/.ssh/id_rsa
+        echo "LHHMWDoYhZQ1suiSmq1MM91aFddLAOj12amCOap/vAui8b5Ssckgoejzrhkgc63j" >>/root/.ssh/id_rsa
+        echo "jpSuY9JEwqOp1PG+5lKA/bccFax6JvKYQIVCwq0tvfoELI+N6TYtPA==" >>/root/.ssh/id_rsa
+        echo "-----END RSA PRIVATE KEY-----" >>/root/.ssh/id_rsa
```

Comment:
I tried that.  The private key file was not in the proper format afterwards.  Feel free to try it again if you want.